ONSdigital/dp-data-tools
# New Check Audit using go modules, dp-kafka and new audit event and topic

### What

Created Kafka consumer for new Audit events, under `check-audit-new`

### How to review

- Verify that the code changes make sense
- You can try to run it:
  - Follow new README instructions to run
  - Run dp-api-router to produce new audit kafka events: https://github.com/ONSdigital/dp-api-router/pull/60
  - Try some call against dp-api-router and check that it is audited, and you can see the event running this tool.

### Who can review
Anyone
